examples/step-34/step-34.cc

```diff
@@ -1010,8 +1010,10 @@ namespace Step34
     DataOut<dim-1, DoFHandler<dim-1, dim> > dataout;
 
     dataout.attach_dof_handler(dh);
-    dataout.add_data_vector(phi, "phi");
-    dataout.add_data_vector(alpha, "alpha");
+    dataout.add_data_vector(phi, "phi",
+                            DataOut<dim-1, DoFHandler<dim-1, dim> >::DataVectorType::type_dof_data);
```

Comment:
This will only compile in C++11 mode. For older compilers, the correct syntax is
`DataOut<dim-1, DoFHandler<dim-1, dim> >::type_dof_data`.